apps/web/src/pages/Pools/PoolAdvertisementPage/PoolAdvertisementPage.tsx

```diff
@@ -796,6 +790,19 @@ export const PoolPoster = ({
                     value={pool.processNumber}
                   />
                 )}
+                {pool.processNumber && (
```

Comment:
Oops, the selection process number field is duplicated.

Reply:
Nice catch! Removed in [Remove duplicate processNumber accordion](https://github.com/GCTC-NTGC/gc-digital-talent/pull/14445/commits/748375122c6eb05c8a0db9d63e6d9188d189fd77)